packages/auth-compat/index.node.ts

```diff
@@ -23,15 +23,10 @@
  */
 export * from './index';
 import { FetchProvider } from '@firebase/auth/internal';
-import {
-  fetch as undiciFetch,
-  Headers as undiciHeaders,
-  Response as undiciResponse
-} from 'undici';
 import './index';
 
 FetchProvider.initialize(
```

Comment:
I'm keeping the FetchProvider around because it's used in our testing framework.